packages/set-version/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@uifabric/set-version",
+  "version": "6.0.0",
+  "description": "Non-Fabric specific package version setter",
```

Comment:
It really doesn't need to be 6 particularly as it has no dependencies on anything from Fabric. I think it can start with 1.

Reply:
1.0.0 it is.